services/brig/test/integration/API/User/Util.hs

```diff
@@ -250,3 +250,16 @@ toAddressBook xs = do
   where
     toCard sha (cardId, entries) = Card (Just $ CardId cardId)
                                         (map (Entry . digestBS sha . T.encodeUtf8) entries)
+
+requestLegalHoldDevice :: Brig -> UserId -> UserId -> LastPrekey -> [Prekey] -> Http ResponseLBS
+requestLegalHoldDevice brig requesterId targetUserId lastPrekey' prekeys = post $ brig
+    . path "/i/clients/legalhold/request"
+    . contentJson
+    . body payload
+  where
+    payload = RequestBodyLBS . encode $ object
+        [ "requester" .= requesterId
+        , "target_user" .= targetUserId
+        , "last_prekey" .= lastPrekey'
```

Comment:
I should double-check consistency here before merging

Reply:
don't the tests do that for you?  and wouldn't it be easier to use the ToJSON instance instead of a Value?